circle.yml

```diff
@@ -75,7 +88,7 @@ jobs:
             docker-compose exec web make -f Makefile-docker update_deps
             docker-compose exec web supervisorctl restart all
             # Start Test in Firefox docker container
-            docker-compose exec selenium-firefox tox -e ui-tests
+            docker-compose exec --user root selenium-firefox tox -e ui-tests
```

Comment:
Hack to fix the test-runs after https://github.com/jrbenny35/selenium-firefox/commit/c853be0de3b84184f9b4417be7b81e9214b5e84d

Reply:
Is that supposed to be permanent ?

Reply:
definitely not, once there's an upstream fix I'll remove it